src/web-ui/src/router/index.js

```diff
@@ -110,6 +110,9 @@ AmplifyEventBus.$on('authState', async (state) => {
       storeUser.sign_up_date = now.toISOString()
       newSignUp = true
     }
+    if (!storeUser.phone_number) {
+      storeUser.phone_number = null
+    }
```

Comment:
This doesn't make sense. Why not always set the phone number to null? Why set it to null in the first place? Deserves a comment at least, if even necessary.

Reply:
Good catch James. I need to remove this piece. This was added as a part of the trial and error when the event from UI was not triggering the campaign.